pkg/kapp/app/recorded_app.go

```diff
@@ -457,11 +435,6 @@ func (a *RecordedApp) labeledApp() (*LabeledApp, error) {
 	return &LabeledApp{sel, a.identifiedResources}, nil
 }
 
-func (a *RecordedApp) isKappApp(app *corev1.ConfigMap) bool {
```

Comment:
Same as above -  just wondering if the assumptions changed here - or why we dropped this

Reply:
Actually I think I answered my own question - we are assuming the suffix has meaning and is a kapp app right?

Reply:
Yep, we are assuming that the suffix is internal and people will not use it.
Also, another reason to remove the `isKappApp` check was to make it consistent with current deploy and rename.
> Remove isKappApp check as we don't have that with existing rename and update as well. If we want to, we can add it in all places later on.